lib/concerns/librarycommon.rb

```diff
@@ -0,0 +1,26 @@
+module Concerns
+  module LibraryCommon
```

Comment:
If your filenames are a combination of more than one word then it should be in snake case. In essence, you'll have `library_common.rb`.